package.json

```diff
@@ -181,6 +196,26 @@
           "command": "peripheral-inspector.svd.forceRefresh",
           "when": "view == peripheral-inspector.svd && viewItem =~ /peripheral.*/"
         },
+        {
+          "command": "peripheral-inspector.svd.exportNode",
+          "when": "view == peripheral-inspector.svd && viewItem == cluster"
+        },
+        {
+          "command": "peripheral-inspector.svd.exportNode",
+          "when": "view == peripheral-inspector.svd && viewItem == registerRW"
+        },
+        {
+          "command": "peripheral-inspector.svd.exportNode",
+          "when": "view == peripheral-inspector.svd && viewItem == register"
+        },
+        {
+          "command": "peripheral-inspector.svd.exportNode",
+          "when": "view == peripheral-inspector.svd && viewItem == registerRO"
+        },
+        {
+          "command": "peripheral-inspector.svd.exportNode",
+          "when": "view == peripheral-inspector.svd && viewItem =~ /peripheral.*/"
```

Comment:
Is there any reason we only allow export of specific nodes? Can we export any?

Reply:
After reviewing the functionality, I found no technical reason to restrict the export to only peripheral.* nodes. I’ve updated the implementation to allow exporting any node in the peripheral-inspector.svd view

Reply:
Great!